lib/api/api_build.dart

```diff
@@ -66,7 +67,7 @@ class CookieInterceptor extends RequestInterceptor {
 }
 
 ChopperClient initChopperClient(CookieStore cookieStore) => ChopperClient(
-    baseUrl: Uri.parse(ConstantDev.hostAddress),
+    baseUrl: ConstantDev.hostAddress,
```

Comment:
Z tego co pamiętam to ta zmiana była potrzebna, aby umożliwić działanie requestów z doklejaniem headera
Żeby to działało, trzeba zaktualizować wersję chopper i chopper_generator na minimum 5.1.0 ( chopper: ^5.1.0, chopper_generator: ^5.1.0+1). Na deploymencie również jest nowsza wersja. Zerknij proszę, czy ta zmiana, którą tu dodałeś jest okej, w razie czego możemy się zdzwonić

Reply:
Racja, zapomniałem zmienić z powrotem